src/Microsoft.Diagnostics.Monitoring.WebApi/Operation/EgressOperation.cs

```diff
@@ -57,11 +78,30 @@ public async Task<ExecutionResult<EgressResult>> ExecuteAsync(IServiceProvider s
             }, logger, token);
         }
 
+        public EgressProcessInfo GetEgressProcessInfo()
```

Comment:
nit: make this an init property: `public EgressProcessInfo ProcessInfo { get; private init; }`

Reply:
I think because one of the $(TestTargetFrameworks) is netcoreapp3.1, we can't use `init` without adding a dummy class to `System.Runtime.CompilerServices` since there's only support for .NET 5+, so I think it might be better to just use `{ get; private set; }`.